runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -120,6 +138,20 @@ public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[]
     @Override
     public <R> R access(ICorfuSMRAccess<R, T> accessMethod,
                         Object[] conflictObject) {
+        R value;
+        boolean isEnabled = MetricsUtils.isMetricsCollectionEnabled();
```

Comment:
That argument exists to keep related counters in sync.  Some fraction of the time, metrics in the field are by people with hair on fire trying to diagnose a problem. Seeing inconsistent values of related metrics is a distraction that they don't need.  That's also why I added the `isMetricsEnabled` argument to @ServerHandler functions.  (I have removed passing that flag to the `stopConditionalContext()` method, there was enough info with context ==/!= null to use 1 arg only.)